apps/forge/flattened/StablePriceOracle-Flattened.sol

```diff
@@ -1363,6 +1369,7 @@ contract StablePriceOracle is IPriceOracle, IERC165, Ownable {
         price3LetterAttoUSDPerSec = (_annualRentPricesUSD[2] * 1e18) / secondsPerYear;
         price4LetterAttoUSDPerSec = (_annualRentPricesUSD[3] * 1e18) / secondsPerYear;
         price5LetterAttoUSDPerSec = (_annualRentPricesUSD[4] * 1e18) / secondsPerYear;
+        price6LetterAttoUSDPerSec = (_annualRentPricesUSD[5] * 1e18) / secondsPerYear;
```

Comment:
Instead of using `_annualRentPricesUSD[x] * 1e18` you can also just use `_annualRentPricesUSD[x] ether`. Does the same thing and is a bit cleaner.

Reply:
Sounds confusing to anyone who doesn't know that keyword **ether** is shorthand for _mult by 1e18_, though. I prefer standard notation, because it's not specific solidity knowledge, and is explicit in the operation. Generally speaking, I prefer explicitness over implicitness for coding style